tests/devhub/test_addon_submissions.py

```diff
@@ -0,0 +1,36 @@
+import pytest
+
+from pages.desktop.developers.devhub_home import DevHubHome
+
+
+@pytest.mark.sanity
+@pytest.mark.serial
+# The first test starts the browser with a normal login in order to store de session cookie
+@pytest.mark.login('submissions_user')
+def test_submit_listed_addon(selenium, base_url, variables, wait):
+    page = DevHubHome(selenium, base_url).open().wait_for_page_to_load()
+    # TODO: add steps for submitting listed addon
+
+
+@pytest.mark.sanity
+@pytest.mark.serial
+@pytest.mark.create_session(
+    'submissions_user'
+)  # starts the browser with an active session (no login needed)
+def test_submit_mixed_addon_versions(selenium, base_url, variables, wait):
+    page = DevHubHome(selenium, base_url).open().wait_for_page_to_load()
+    # TODO: add steps for submitting addon new version
+
+
+# TODO: more tests that use @pytest.mark.create_session go here
+
+
+# This last part handles the session cookie invalidation and user file deletion
+@pytest.mark.sanity
+@pytest.mark.serial
+@pytest.mark.create_session('submissions_user')
+@pytest.mark.clear_session
+def test_clear_session(selenium, base_url):
+    with pytest.raises(FileNotFoundError):
+        with open('submissions_user.txt', 'r') as file:
+            file.read()
```

Comment:
This should just be moved to the selenium fixture after a ```yield``` statement. See: https://docs.pytest.org/en/6.2.x/fixture.html#yield-fixtures-recommended

Reply:
Ok, I've managed to integrate this part inside the selenium fixture after a `yield` statement, but I still had to assign it to a marker, as before - see https://github.com/mozilla/addons-release-tests/pull/475/files#diff-e52e4ddd58b7ef887ab03c04116e676f6280b824ab7469d5d3080e5cba4f2128R121-R149
This allows me to call upon the clean-up part on demand, otherwise it would have run after each test, and I need the authentication cookie to be active for the duration of the test session in most cases.

I've also changed the clean up part to use the `DELETE` session API directly, since it is a bit faster than a logout from the browser with selenium.

Reply:
Why a marker? You could just check the file path and if there's a file delete and if not, do nothing.

Reply:
Because the file will be there for the entire duration of the run - which is what I need. If the fixture deletes the file before my test session ends, than all the other tests will fail, since they rely on that file being there. Note, that the file is created at the start of the run, through a normal browser login, then, all the subsequent tests will rely on that file for authentication. That's in short what the whole set-up is supposed to do.

Reply:
So it seems that you need to change the scoping of the fixtutre. I don't like how there's a "test" to just delete a file.

Reply:
I no longer need to write a test to delete the file with the current set-up. As I mentioned, the fixture will be called on the last test that runs in the suite, which does it's own thing - like verifying that an add-on is approved for example - and once the test steps are complete, the clean-up part will do it's thing without having to specifically add that in the test. 

I've tried to give an example of how this would work here: 
https://github.com/mozilla/addons-release-tests/pull/475/files#diff-1c11238e8621a678242a54db8a04209acc4dcf4a79ab7c22cb7ca62c86111b23R28-R39